Sming/SmingCore/Timer.h

```diff
@@ -12,16 +12,10 @@
 #define _SMING_CORE_Timer_H_
```

Comment:
The guard should be in uppercase letters. I see now from where comes the "bad" influence.